INSTALLATION.md

```diff
@@ -0,0 +1,19 @@
+# Installation
+
+## Prerequisites
+
+- Node.js LTS Dubnium or newer
```

Comment:
Can we tell people to use [nvm](https://github.com/creationix/nvm) so this no longer contains the node version. We just need to get them to run `nvm use` in the directory